bad_validation_test.py

```
@@ -0,0 +1,45 @@
+    print("Received email:", email)
+
+    # ❌ No validation: should check if name/email start with special characters
+    # Expected: check name[0].isalnum() before saving
+
+    user_data = {
+        "name": name,
+        "email": email
+    }
+
+    # ❌ Uses eval() – very risky, disallowed
+    result = eval("2 + 2")
```

Ruff (0.14.4)

Use of possibly insecure function; consider using `ast.literal_eval`

(S307)